src/index.ts

```diff
@@ -56,9 +53,18 @@ export default class ServerlessConventions {
 
     // Make sure `ignore` is defined in case `serverless.yml` has `conventions` with an empty `ignore` block
     this.conventionsConfig.ignore = this.conventionsConfig.ignore || {};
+
+    // Notify user if they are using unsupported Serverless version
+    const version = parseFloat(this.serverless.getVersion());
+    if (version < 3) {
+      this.log.warning(
+        `You are using an old Serverless version (${version}).\n` +
```

Comment:
I think `version` here should be the original response from `getVersion`. Otherwise you might be missing a minor version number